quiz.js

```diff
@@ -0,0 +1,55 @@
+var treeArray = [];
+var pineTree = {
+	treeCharacter: "", 
+	numHeight: ""
+};
+
+var myButton = document.getElementById('thisWillGrowATree');
+myButton.addEventListener('click', getValues);
+window.addEventListener('onkeyup', inputKeyUp);
+
+function growTree(height) {
+	var treeWidth  = height.treeCharacter;
+	var treeHeight = height.numHeight;
+	var emptySpace = " ";
+	var centered = Math.ceil(height.numHeight / 2);
+	console.log(emptySpace.repeat(centered + parseInt(height.numHeight)/2 - 1),treeWidth);
+	
+	
+
+	for (var i = 0; i <= height.numHeight - 2; i++) {
+		treeWidth = treeWidth + height.treeCharacter + height.treeCharacter;// + height.numHeight;
+		console.log(emptySpace.repeat((centered - i) + parseInt(height.numHeight) / 2 - 1) + treeWidth);
+
+		}
+}
+
+function inputKeyUp(event) {
+	console.log("keyWasPressed", event);
+    if(inputKeyUp.keyCode === 13) {
+    	console.log("height", event);
+    	growTree(treeArray);
+    }    
+}
+function listenForClick() {
+	growTree(treeArray);
+}
+
```

Comment:
It looks like your keyup events aren't working. You're suppose to be able to hit enter to pass an object into your make tree function. Currently trying to pass in an array by the looks.